react/Button/index.jsx

```diff
@@ -56,8 +56,8 @@ export const ButtonLink = props => {
 // Proptypes
 const commonPropTypes = {
   children: PropTypes.node,
-  label: PropTypes.string,
-  icon: PropTypes.string,
+  label: PropTypes.node,
+  icon: PropTypes.node,
```

Comment:
For my own culture, when the PropType is `node` but you send a `string` it doesn't "break" because a string can be related to a text node, is that it?

Reply:
https://reactjs.org/docs/typechecking-with-proptypes.html

Reply:
OK TIL.
funny, out of instinct, I was expecting `node` to be elements or even text node but an array!? 
Well ok then. 
Thank you. 🙏